README.md

```diff
@@ -1 +1,188 @@
-# Portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" /> -->
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
```

Comment:
- [ ] Kindly remove this part from your README file as it is currently of no use in this project :+1: